index.html

```diff
@@ -2260,6 +2260,84 @@ <h3>Role Mapping Table</h3>
           <p><sup>[Note 2]</sup> This specification does not currently contain guidance for when user agents should fire system alert events. Some guidance may be added to the specification at a later date but it will be a recommendation (SHOULD), not a requirement (MUST).</p>
         </div>
       </section>
+
+
+      <section id="mapping_generic_role">
+        <h3>Role Mapping for Generic Host Language Elements</h3>
+        <p>ARIA 1.2 introduced a new <a class="role-reference" href="#generic"><code>generic</code></a> role, intended for host language elements without a specific or consistent user interface semantic meaning.</p>
+        <p>Generic elements are often used by web authors for a variety of custom grouping, stylistic, and functional containers. Examples of such elements include <a href="https://w3c.github.io/html-aam/#el-div"><code>&lt;div&gt;</code></a> and <a href="https://w3c.github.io/html-aam/#el-span"><code>&lt;span&gt;</code></a> in HTML, and <a href="https://www.w3.org/TR/svg-aam-1.0/#details-id-40"><code>&lt;g&gt;</code></a> in SVG. Earlier versions of HTML contained many generic elememts associated with a default style renderings, such as <code>&lt;b&gt;</code>, <code>&lt;i&gt;</code>, and <code>&lt;blink&gt;</code>.</p>
```

Comment:
```suggestion
        <p>Generic elements are often used by web authors for a variety of custom grouping, stylistic, and functional containers. Examples of such elements include <a href="https://w3c.github.io/html-aam/#el-div"><code>&lt;div&gt;</code></a> and <a href="https://w3c.github.io/html-aam/#el-span"><code>&lt;span&gt;</code></a> in HTML, and <a href="https://www.w3.org/TR/svg-aam-1.0/#details-id-40"><code>&lt;g&gt;</code></a> in SVG. Earlier versions of HTML contained many generic elements associated with a default style renderings, such as <code>&lt;b&gt;</code>, <code>&lt;i&gt;</code>, and <code>&lt;blink&gt;</code>.</p>
```
Elememts is kind of cute tho :)